website/server/controllers/api-v3/user.js

```diff
@@ -28,7 +28,29 @@ let api = {};
  * @apiGroup User
  *
  * @apiSuccess {Object} data The user object
- */
+ *
+ * @apiDescription The user profile contains data related to the authenticated user including (but not limited to);
+ * Achievements
+ * Authentications (including types and timestamps)
```

Comment:
on the other hand, I'm looking at what MathWiz did with Content, I could just empty out most of the sub-keys like below... is that too little information?

```{
  "success": true,
  "data": {
    "_id": "11be077e-4a54-4d3a-96b6-8f14c101aeff",
    "migration": "20170131_habit_birthday.js",
    "loginIncentives": 9,
    "webhooks": [],
    "pushDevices": [],
    "extra": {},
    "tasksOrder": {
      "rewards": [ ],
      "todos": [ ],
      "dailys": [],
      "habits": [ ]
    },
    "inbox": {},
    "tags": [],
    "notifications": [    ],
    "stats": {
        --stats, training, buffs--    },
    "profile": {
      "name": "Sisyphus"
    },
    "preferences": {},
    "party": {},
    "guilds": [],
    "invitations": {},
    "challenges": [],
    "newMessages": {},
    "lastCron": "2017-02-06T12:24:56.080Z",
    "items": {},
    "history": {},
    "flags": {},
    "purchased": { },
    "balance": 0,
    "contributor": {},
    "backer": {},
    "achievements": {},
    "_v": 286,
    "auth": { },
    "id": "11be077e-4a54-4d3a-96b6-8f14c101aeff"},
  "notifications": []
}